frontend/src/components/common/SchedulingProgressBar.tsx

```diff
@@ -0,0 +1,28 @@
+import { Flex } from "@chakra-ui/layout";
+import { Step, Steps } from "chakra-ui-steps";
+import React from "react";
+
+import { SchedulingProgessBarProps } from "../pages/Scheduling/types";
+
+const steps = [{ label: "1" }, { label: "2" }, { label: "3" }, { label: "4" }];
+
+const SchedulingProgressBar = ({
+  activeStep,
+}: SchedulingProgessBarProps): JSX.Element => {
+  return (
+    <Flex flexDir="column" width="100%">
+      <Steps
+        activeStep={activeStep}
+        responsive={false}
+        colorScheme="teal"
+        size="sm"
```

Comment:
```suggestion
        size="sm"
        checkIcon={}
```
We can pass in a circle icon or something from Chakra